output/components.eslint.txt

```diff
@@ -18,6 +18,9 @@ Scoping to packages that match 'react-talend-components'
   2:1  error  '@kadira/react-storybook-addon-info' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
   7:2  error  Unexpected require()                                                                                      global-require
 
+/home/travis/build/Talend/ui/packages/components/src/List/DisplayTable/DisplayTable.component.js
+  26:31  error  Expected '!==' and instead saw '!='  eqeqeq
```

Comment:
Yep. I'm also in favor of removing the ternary.
I'll see what I can do. Maybe go back to a good old if statement ;)